mobile/status.go

```diff
@@ -1066,6 +1069,32 @@ func writeHeapProfile(dataDir string) string { //nolint: deadcode
 	return makeJSONResponse(err)
 }
 
+// StartMutexProfiling starts mutex profiling and HTTP server for pprof
+func StartMutexProfiling(address string) string {
+	return callWithResponse(startMutexProfiling, address)
+}
+
+func startMutexProfiling(address string) string {
+	runtime.SetMutexProfileFraction(5)
+
+	mux := http.NewServeMux()
+
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/mutex", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+
+	go func() {
+		if err := http.ListenAndServe(address, mux); err != nil {
+			logutils.ZapLogger().Error("failed to start pprof server", zap.Error(err))
+		}
+	}()
+
+	return makeJSONResponse(nil)
+}
```

Comment:
Please reuse this:
https://github.com/status-im/status-go/blob/2ced83abb7ef30762fd8e2f8a35ca90ba1163df2/profiling/profiler.go#L22-L37

Which is used like this in `statusd`:
https://github.com/status-im/status-go/blob/e477269983e3baa3aac5445dff44441b9a70a782/cmd/statusd/main.go#L318-L321

Perhaps it's better to pass full `address` as you do, rather than just the `port`.